src/ngx_http_lua_log.c

```diff
@@ -227,12 +237,17 @@ log_wrapper(ngx_log_t *log, const char *ident, ngx_uint_t level,
 
                 break;
 
+            case LUA_TTABLE:
+                luaL_callmeta(L, i, "__tostring");
+                q = (u_char *) lua_tolstring(L, -1, &len);
+                p = ngx_copy(p, q, len);
+                break;
+
             case LUA_TLIGHTUSERDATA:
                 *p++ = 'n';
                 *p++ = 'u';
                 *p++ = 'l';
                 *p++ = 'l';
-
```

Comment:
@thibaultCha This change is unrelated to this PR, better exclude it :) But I'm fine with including it anyway.